client/src/Hooks/useAxios.js

```diff
@@ -0,0 +1,20 @@
+import {useState, useEffect} from 'react'
+import axios from 'axios'
+
+const useAxios = url => {
+    const [data, setData] = useState([])
+
+    useEffect(()=> {
+        axios
+            .get(url)
+            .then(results => {
+                setData(results.data)
+            })
+            .catch(error=> {
+                console.log(error)
+            })
+    },[url])
+    return [data]
+}
+
+export default useAxios
```

Comment:
this is a  really cool custom hook. good job making one not giving to everyone in the training kit.